service/status-report/main.go

```diff
@@ -166,3 +205,17 @@ func main() {
 		klog.Exitf("Failed to update mon configmap for storageClient %v: %v", storageClient.Status.ConsumerID, err)
 	}
 }
+
+func getVersionFromChannel(channel string) (*version.Version, error) {
+	_, versionFromChannel, found := strings.Cut(channel, "-")
+	if !found {
+		return nil, fmt.Errorf("unable to refer version from channel name")
+	}
+
+	genericVersion, err := version.ParseGeneric(versionFromChannel)
+	if err != nil {
+		return nil, fmt.Errorf("unable to parse referred version from channel: %w", err)
+	}
+
+	return genericVersion, nil
+}
```

Comment:
[nb-ohad](https://github.com/nb-ohad) [Mar 15, 2024](https://github.com/red-hat-storage/ocs-client-operator/pull/89#discussion_r1525442586)

I don't see a need to extract the version from the channel name